extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -406,19 +406,19 @@ export class AmpIframe extends AMP.BaseElement {
     }
 
     if (newHeight !== undefined || newWidth !== undefined) {
-      this.attemptChangeSize(newHeight, newWidth, () => {
+      this.attemptChangeSize(newHeight, newWidth).then(() => {
         if (newHeight !== undefined) {
           this.element.setAttribute('height', newHeight);
         }
         if (newWidth !== undefined) {
           this.element.setAttribute('width', newWidth);
         }
-      });
-    } else {
-      user.error(TAG_,
+      }, () => {
+        user.error(TAG_,
           'ignoring embed-size request because'
```

Comment:
in this case it is correct - because it is in the else bloc of 
`if (newHeight !== undefined || newWidth !== undefined)`